src/reducers/components.test.js

```diff
@@ -58,4 +58,14 @@ describe('components.js', () => {
       }
     });
   });
+
+  it('should handle an action of type COMPONENT_REMOVE', () => {
+    const state = { testId: {}, testId2: {} };
+    const action = { type: 'COMPONENT_REMOVE', payload: { id: 'testId' } };
```

Comment:
At least for this file, can we open up the curly braces instead of putting everything on the same line? Eventually as the project grows we can add prettier.

Reply:
Yeah, I agree that this one would be more readable if opened up.